packages/suite/src/actions/wallet/sendFormActions.ts

```diff
@@ -146,7 +133,7 @@ export const convertDrafts = () => (dispatch: Dispatch, getState: GetState) => {
         const conversionToUse =
             areSatsSelected && areSatsSupported ? amountToSatoshi : formatAmount;
 
-        const updatedDraft = clone(draft);
+        const updatedDraft: FormState = window.structuredClone(draft);
```

Comment:
Did you checked output in /lib that this is really correctly transpiled and polyfilled?